layers/+lang/typescript/packages.el

```diff
@@ -30,7 +30,9 @@
     :modes typescript-mode typescript-tsx-mode))
 
 (defun typescript/pre-init-eldoc ()
-  (spacemacs|use-package-add-hook tide :post-config (eldoc-mode)))
+  (spacemacs|use-package-add-hook tide :post-init
+                           (add-hook 'typescript-tsx-mode-hook 'eldoc-mode t)
+                           (add-hook 'typescript-mode-hook 'eldoc-mode t)))
```

Comment:
Can't use `spacemacs/add-to-hooks` here because we need to append. Should I add an optional append flag to `spacemacs/add-to-hooks`?

Reply:
Yes it would be a good addition, for now I merge your PR as it fixes my mistake.